tests/conftest.py

```diff
@@ -50,3 +50,9 @@ def config(request: Any) -> Image.Config:
         tools_tree_distribution=cast(Distribution, request.config.getoption("--tools-tree-distribution")),
         debug_shell=request.config.getoption("--debug-shell"),
     )
+
+
+@pytest.fixture(autouse=True)
+def github_ci_sections(request: Any) -> Any:
```

Comment:
This should be named `test_ci_sections`

Reply:
I went with just `ci_sections`, since it's not a test.